app/services/dataworks_mappers/open_alex.rb

```diff
@@ -0,0 +1,147 @@
+# frozen_string_literal: true
+
+module DataworksMappers
+  # Map from OpenAlex metadata to Dataworks metadata
+  class OpenAlex < Base # rubocop:disable Metrics/ClassLength
+    IDENTIFIER_TYPES = {
+      'openalex' => 'OpenAlex',
+      'doi' => 'DOI'
+    }.freeze
+
+    def perform_map
+      {
+        identifiers:,
+        titles:,
+        creators:,
+        publication_year: source[:publication_year].to_s,
+        url: source.dig(:primary_location, :landing_page_url),
+        access:,
+        provider: 'OpenAlex',
+        dates:,
+        language: source[:language],
+        rights_list:,
+        subjects:,
+        funding_references:,
+        related_identifiers:
+      }.compact_blank
+    end
+
+    private
+
+    def access
+      return 'Restricted' unless source.dig(:primary_location, :is_oa)
+
+      'Public'
+    end
+
+    def affiliations_for(institutions)
+      return unless institutions
+
+      Array(institutions).map do |institution|
+        {
+          affiliation_identifier: institution[:ror],
+          affiliation_identifier_scheme: 'ROR',
+          name: institution[:display_name]
+        }.compact
+      end
+    end
+
+    def creators
+      return unless source[:authorships]
+
+      Array(source[:authorships]).map do |author|
+        {
+          name: author.dig(:author, :display_name),
+          name_identifiers: name_identifiers_for(author[:author]),
+          affiliation: affiliations_for(author[:institutions])
+        }.compact_blank
+      end
+    end
+
+    def dates
```

Comment:
These particular properties have to do with when the record is created in openalex and not the dataset itself. Could you please remove these particular mappings? Thank you